### ISSUE TITLE
Update: April 9, 2018

### ISSUE THREAD

**Issue by kaipak** [pangeo-data/storage-benchmarks#24]
We've worked out most of the base code and have some test results. Need to clean up some things like settling in on size of datasets, units, making plots more legible, etc. I plan on running a suite of tests on GCC shortly and will start publishing this week. 

Related to this, @jreadey and I are working on some more detailed docs so the use cases and tests are more clearly defined. Several more have cropped up in various other threads and keeping track of all of them continues to be a challenge. I'm hoping that being more prescriptive about will initiate further discussion.

Here's some preliminary runs from my laptop. Upload speeds to GCS are predictably bad as these were run from my home. I have early results from GCC as well and will combine them shortly. https://kaipak.github.io/storage-benchmarks

**Comment by rabernat** [comment]
This is great Kai! It would be great to have a key to help decipher what the various benchmarks represent.

**Comment by kaipak** [comment]
> `This is great Kai! It would be great to have a key to help decipher what the various benchmarks represent`.

Yes, that is one of our top things to fix. It's completely opaque how big these datasets are and I plan on cleaning this stuff up.

**Comment by kaipak** [comment]
I was able to upload some preliminary results from GCC for Xarray tests, but these are relatively small datasets and not indicative of full tests that will be run. I mainly wanted to figure out the workflow for combining results from different environments and publishing.

![screen shot 2018-04-09 at 6 53 56 pm](https://user-images.githubusercontent.com/9453271/38527197-891e2a40-3c27-11e8-9545-dd0140f10104.png)

As you can see, I still need to work on the plots to make them more meaningful. FYI, the random data sets are 256x512x5 which is roughly 10.5 megabytes. Also, blue is GCP and red is my laptop. You can click on elements on the plot pages so it can be easier to visualize and compare.
